components/brave_rewards/resources/ui/components/mobile/promoMobile/promo_mobile.style.ts

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import palette from 'brave-ui/theme/colors'
+
+interface StyleProps {
+  src?: string
+}
+
+export const root = styled('div')<{}>`
+  max-width: 373px;
+  width: 373px;
+  height: 338px;
+  overflow: hidden;
+  margin-bottom: 12px;
+`
+
+export const closeIcon = styled('div')<{}>`
+  width: 12px;
+  height: 12px;
+  color: #fff;
+  float: right;
+  margin: 11px 11px 0 0;
+`
+
+export const background = styled('div')<StyleProps>`
```

Comment:
Is there a way to do this without these styled components custom props? I'd like us to stick with plain CSS if possible.

One way to do this is to set a `className` on this `style.background` component, and then in the style props use that class name to set the actual image URL.

```tsx

import bitflyerVerificationBg from '../assets/bitflyer_verification_bg.png'

// ...

export const background = styled.div`
  &.bitflyer-verification-promo {
      background: url('/${bitflyerVerificationBg}' no-repeat;
  }
  ...
`
```

One advantage of this approach is that the component gets to define what images it will display as backgrounds. The downside is that there is a little more wire-up.

Reply:
Great approach, fixed.